js/src/lib/Details/Markdown.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import marked from 'marked';
 import xss from 'xss';
-import hljs from 'highlight.js';
+import hljs from '../../../../_js/highlight/build/highlight.pack';
```

Comment:
It's probably simpler if we just include this file, WDYT @Daniel15 ?

Reply:
I'll try to split the code later, but I didn't understand really why to do that